libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -192,6 +192,11 @@ struct MqttPublishInfo
      */
     bool retain;
 
+    /**
+     * @brief Whether this is a duplicate publish message.
+     */
+    bool dup;
```

Comment:
Ok. Assume it is due to the target C std change to ANSI C instead of C99. Will handle it as a separate change as it may not be just related to Publish. Thanks